scripts/reload-clients

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+import argparse
+import configparser
+import logging
+import os
+import sys
+import time
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+from scripts.lib.setup_path import setup_path
+
+setup_path()
+
+import requests
+from requests.adapters import HTTPAdapter
+from urllib3.util import Retry
+
+from scripts.lib.zulip_tools import get_config, get_config_file, get_tornado_ports
+
+config_file = get_config_file()
+reload_rate = int(
+    get_config(
+        config_file,
+        "application_server",
+        "client_reload_rate",
+        "50",
+    )
+)
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--rate", type=int, help="Number of clients to reload per second", default=reload_rate
+)
+
+args = parser.parse_args()
+reload_rate = args.rate
+
+secret_config_file = configparser.RawConfigParser()
+secret_config_file.read("/etc/zulip/zulip-secrets.conf")
+shared_secret = get_config(secret_config_file, "secrets", "shared_secret")
+assert shared_secret
+
+# Perform retries with backoff, including on POST requests.  Failure
+# to send this request successfully means that clients may fail to
+# reload, so we want to be somewhat resilient to failures.  Since we
+# are on localhost, we do not expect network failures, only Tornado
+# restarts, to cause failures here.
+retry = Retry(total=3, backoff_factor=0.1, allowed_methods=Retry.DEFAULT_ALLOWED_METHODS | {"POST"})
+c = requests.Session()
```

Comment:
Hmm, I'm not sure we want fast retries on this sort of thing -- if the Tornado server is somehow returning an error late in the request handler, after having actually sent a bunch of reload events, retries could lead to going much faster than intended?

I guess a limit of 3 retires makes that pretty safe though.

Reply:
Adjusted to 2s, 4s, 8s.